tests/shell/026_init_no_masterfiles.sh

```diff
@@ -0,0 +1,11 @@
+set -e
+set -x
+cd tests/
+mkdir -p ./tmp/
+cd ./tmp/
+touch cfbs.json && rm cfbs.json
+rm -rf .git
+
+cfbs --non-interactive init --masterfiles=no
+!( grep '"name": "masterfiles"' cfbs.json )
+cfbs build
```

Comment:
TODO: We should probably add a warning or error when building an empty project.

Reply:
[CFE-4090](https://tracker.mender.io/browse/CFE-4090)